src/agent/wrapped-agent.ts

```diff
@@ -139,4 +141,24 @@ export class WrappedAgent<T extends HttpAgent> implements HttpAgent {
         this.agent.prependOnceListener(eventName, listener);
         return this;
     }
+
+    createConnection(options: NetConnectOpts, callback?: (err: Error | null, stream: Duplex) => void): Duplex {
```

Comment:
or is the actual fix those new methods?

Reply:
Partially, the linked `@types/node` PR adds those methods to the `HttpAgent` interface, so we need to implement those here to `implement` the interface correctly (so that TS doesn't throw compilation errors).